gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterManager.java

```diff
@@ -150,6 +150,10 @@ public GobblinClusterManager(String clusterName, String applicationId, Config co
 
     this.applicationId = applicationId;
 
+    //Helix uses System#getProperty() for ZK configuration overrides such as sessionTimeout. The overrides specified
```

Comment:
Since this is useful for any configuration, let's fix the comment which gives me impression that only ZK configuration's overrides are loaded from system configuration.

Reply:
Modified the comment.